src/main/java/net/snowflake/client/core/StmtUtil.java

```diff
@@ -597,7 +597,7 @@ protected static String getQueryResult(String getResultPath, StmtInput stmtInput
    * @throws SFException exception raised from Snowflake components
    * @throws SnowflakeSQLException exception raised from Snowflake components
    */
-  protected static JsonNode getQueryResultJSON(String queryId, SFSession session)
+  public static JsonNode getQueryResultJSON(String queryId, SFSession session)
```

Comment:
This is the utility API to get the Json result for the multiple statement.

Reply:
The rest API end point is:
  private static final String SF_PATH_QUERY_RESULT = "/queries/%s/result";